raiden-dapp/tests/unit/store/notifications/mutations.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { mutations } from '@/store/notifications/mutations';
+import { TestData } from '../../data/mock-data';
+import {
+  NotificationPayload,
+  NotificationsState,
+} from '@/store/notifications/types';
+
+describe('notifications store mutations', () => {
+  const notification = TestData.notifications;
+
+  const createNotificationState = (
+    newNotifications = false
+  ): NotificationsState => {
+    return {
+      notifications: { 1: notification },
+      newNotifications,
+    };
+  };
+
+  test('can delete notification', () => {
+    const state = createNotificationState();
+    const id = TestData.notifications.id;
+
+    mutations.notificationDelete(state, id);
+    expect(state.notifications).toMatchObject({});
+  });
+
+  test('can set notifications to viewed', () => {
+    const state = createNotificationState(true);
+
```

Comment:
Please make sure it was `true` before and this test case does not "accidentally" succeed by previous test cases or similar.

Reply:
Fixed it.